### DISCUSSION TITLE
Mark a chat as read and webhook on read message

### DISCUSSION THREAD

**Discussion by krugerrgabriel** [WhiskeySockets/Baileys#346] [General Questions]
Hello, how are you?

I migrated from a third-party platform to Baileys yesterday and I missed some features.
The first one is marking all messages of a specific chat as read, so that when I send a message to a chat, all its messages are marked as read.
The second one is that when someone reads a message I send, a webhook notification arrives for me to know.

I appreciate it in advance.

**Comment by Riders004** [comment]
> Hello, how are you?
> 
> I migrated from a third-party platform to Baileys yesterday and I missed some features. The first one is marking all messages of a specific chat as read, so that when I send a message to a chat, all its messages are marked as read. The second one is that when someone reads a message I send, a webhook notification arrives for me to know.
> 
> I appreciate it in advance.

Can you provide me that library link

**Reply by krugerrgabriel** [reply]
Here it is 😉
https://github.com/salman0ansari/whatsapp-api-nodejs
